modules/caddyhttp/templates/tplcontext.go

```diff
@@ -17,6 +17,9 @@ package templates
 import (
 	"bytes"
 	"fmt"
+	"github.com/Masterminds/sprig/v3"
+	"github.com/caddyserver/caddy/v2"
+	"github.com/dustin/go-humanize"
```

Comment:
I think your IDE moved these import lines. I don't think it should be done this way, they should be grouped by source, so all stdlib first then github ones together, etc. Running `gofmt -s -w` can fix this up I think.

Reply:
It didn't, but I'll just copy paste the imports from the unmodified file